src/server/worldserver/Master.cpp

```diff
@@ -141,10 +141,10 @@ int Master::Run()
     if (!pidFile.empty())
     {
         if (uint32 pid = CreatePIDFile(pidFile))
-            sLog->outString("Daemon PID: %u\n", pid);
+            sLog->outError("Daemon PID: %u\n", pid); // outError for red color
```

Comment:
Could you please also add this comment here?
https://github.com/azerothcore/azerothcore-wotlk/blob/4a8f1de5381d1cf53cdcc4c5bb7d9ef7c99a8a8f/src/server/game/World/World.cpp#L2005

Reply:
+1

Reply:
we don't have anything like that already?? @Stoabrogga 

btw I couldn't find any error message about "can not bind to 0.0.0.0:port" for the worldserver (i'll try to see what happens to run two worldservers once i update my DB, today or tomorrow)

Reply:
ah ok didn't understand, yes I will do it